src/eventEmitter/index.js

```diff
@@ -0,0 +1,196 @@
+var EventEmitter;
```

Comment:
Looking from the entire code below, this global variable appears to have no use at all and should be removed.